Updated skinning example and text (#64)
KhronosGroup/glTF-Tutorials: gltfTutorial/gltfTutorial_020_Skins.md

    Vertex 8:  0, 1, 0, 0,
    Vertex 9:  0, 1, 0, 0,

> This means that every vertex may be influenced by joint 0 and joint 1  (except for the two vertices at the bottom, which may only be influenced by joint 0). The last 2 components of each vector are ignored here. If there were multiple joints, then one entry of this accessor could, for example, contain

Reviewer: ```suggestion
This means that every vertex may be influenced by joint 0 and joint 1, except the first two vertices are influenced only by joint 0, and the last two vertices are influenced only by joint 1. The last 2 components of each vector are ignored here. If there were multiple joints, then one entry of this accessor could, for example, contain
```

Reviewer: Just a suggestion for how to refer to the unique vertices.

Author: I struggled to find the best wording here, because someone might read these numbers, and wonder _what they mean_. And from the joint numbers alone, it is not clear why the last vertices should *only* be influenced by joint 1. That only becomes clear when taking the _weights_ into account.
